src/pynxtools/dataconverter/writer.py

```diff
@@ -117,11 +116,13 @@ def handle_dicts_entries(data, grp, entry_name, output_path, path):
     - Concatenate dataset in one virtual dataset
     - Internal links
     - External links
-    - compression label"""
+    - compression label
+    """
+    file = None
     if "link" in data:
         file, path = split_link(data, output_path)
     # generate virtual datasets from slices
-    if "shape" in data.keys():
+    if file is not None and "shape" in data.keys():
```

Comment:
I don't understand, how is this related to the issue?

Reply:
Removed!